jwst/background/background_step.py

```diff
@@ -58,13 +58,38 @@ def process(self, input, bkg_list):
                 result = background_sub.subtract_wfss_bkg(
                                 input_model, bkg_name, wlrange_name)
                 result.meta.cal_step.back_sub = 'COMPLETE'
-
             else:
+                # check if input data is NRS_IFU
+                tolerance = 1.0e-15
```

Comment:
Looking at the GWA* values in many of the NIRSpec datasets I have laying around it appears that these keyword values typically have between 6-9 significant digits and when they vary from one exposure to another they vary at around the level of 1.e-3 or 1.e-4. So I would set our starting tolerance somewhere around 1.0e-7 or 1.0e-8.